src/main/scala/com/ing/wbaa/airlock/sts/service/db/MariaDb.scala

```diff
@@ -11,9 +12,17 @@ import scala.util.{ Failure, Success, Try }
 
 trait MariaDb extends LazyLogging {
 
+  protected[this] implicit def system: ActorSystem
+
   protected[this] def mariaDBSettings: MariaDBSettings
 
-  protected[this] implicit def executionContext: ExecutionContext
+  protected[this] implicit lazy val dbExecutionContext: ExecutionContext =
+    Try {
+      system.dispatchers.lookup("db-dispatcher")
+    } match {
+      case Success(dispatcher) => dispatcher
+      case Failure(_)          => system.dispatcher
```

Comment:
`log.warn ..` would be helpful that the sts works on default dispatcher  + the error info `why`